### PR TITLE
Communities of practice page styling

### LINKED ISSUE #1333: Adjust the styling of the Communities of Practice page
### Overview
We need to make the [Communities of Practice page](https://www.hackforla.org/communities-of-practice) body background beige/pink and decrease the size of the subheading font so that it will be consistent with the rest of the website and design system.

### Action Items
- [ ] View the [Figma file of the Communities of Practice page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3194%3A25627) for reference (final design in pink rectangle)
- [ ] Adjust the page card subheadings in **desktop** to 24px (instead of 28px) in the [Communities of Practice page](https://www.hackforla.org/communities-of-practice) - use rem unit
- [ ] Adjust the page card subheadings in **mobile** to 20px (instead of 22px) - use rem unit
- [ ] Turn the body background color of the page from gray to the beige/pink background color we use on the homepage, guide pages, etc. (but leave the header white)
- [ ] In mobile- add spacing between Meeting Times and Leadership Type sections by putting them in separate p tags
   - [ ] This will affect the desktop version as well- make sure everything still looks good (or minimize the margin between these lines in the desktop version to make it look similar to the Figma design)

### Resources/Instructions
[Communities of Practice page](https://www.hackforla.org/communities-of-practice)
[Homepage](https://www.hackforla.org/#) (for reference- beige background)

<details>
<summary>Subheadings:</summary>
<br>
Screen Shot 2021-03-26 at 11 23 48 PM
</details>

<details>
<summary>Background color:</summary>
<br>

How page looks now:

Screen Shot 2021-03-24 at 2 15 41 AM

How it should look:

Screen Shot 2021-03-24 at 2 15 32 AM</details>

<details>
<summary>Meeting Times/Leadership Type sections:</summary>
<br>

How it looks now in mobile:

Screen Shot 2021-03-27 at 4 15 46 PM

How it should look:

Screen Shot 2021-03-27 at 4 15 51 PM

</details>

### PR DESCRIPTION
**Pull request by evancchen**
Fixes #1333 
Styled communities of practice page

### CODE REVIEW

File: _sass/components/_communities-of-practice.scss [hunk 1]
```diff
@@ -71,7 +71,7 @@
 }
 
 .content--communities {
-  background: #e5e5e5;
+  background: #f7f5f5;
```
Comment (akibrhast) [inline _sass/components/_communities-of-practice.scss]: @evancchen Pleas replace raw color with variable name `$color-pink`

**Review by akibrhast — CHANGES_REQUESTED** [review]

**Comment by akibrhast** [comment]
@evancchen Please provide a screenshot of what your affected changes look like.

**Comment by evancchen** [comment]
image
The page card subheading is now smaller, the body background color is `$color-pink`, and there is spacing between Meeting Times and Leadership Type

**Review by akibrhast — APPROVED** [review]
LGTM

**Pull request merged**